lebedigital/calibration/calibrationWorkflow.py

```diff
@@ -0,0 +1,202 @@
+    # ========================================
+    # "uninformed" Normal prior for the E modulus
+    loc_E = 100  # kN/mm2
+    scale_E = 100  # this can be inferred too
+
+    # Uniform prior for experimental noise/model discrepancy
+    low_sigma = 0
+    high_sigma = 0.005
+
+    # =========================================
+    #       Define the Inference Problem
+    # =========================================
+
+    # initialize the inverse problem with a useful name
+    problem = InverseProblem("compression test calibration")
+
+    # add all parameters to the problem
+    problem.add_parameter('E', 'model',
+                          tex="$E$",
+                          info="Slope of the graph",
+                          prior=('normal', {'mean': loc_E,
+                                            'std': scale_E}))
+
+    problem.add_parameter('sigma', 'likelihood',
+                          tex=r"$\sigma$",
+                          info="Std. dev, of 0-mean noise model",
+                          prior=('uniform', {'low': low_sigma,
+                                             'high': high_sigma}))
+
+    # Load the forward model and add it to the Inverse problem
+    linear_elasticity = LinearElasticityCylinder("linear_elasticity_cylinder")
+    problem.add_forward_model(linear_elasticity)
+
+    # ============================================
+    #     Add test data to the Inverse Problem
+    # ============================================
+
+    y_test = exp_output['stress'] * (np.pi * (exp_output['diameter'] / 2) ** 2)  # in kN
+    # add the experimental data
+    problem.add_experiment(
+        experiment_name,
+        fwd_model_name="linear_elasticity_cylinder",
+        sensor_values={
+            'nu': 0.2,
+            'height': exp_output['height'],
+            'radius': exp_output['diameter'] / 2,
+            'displacement_list': exp_output['displacement'],
+            linear_elasticity.output_sensor.name: y_test,
+        },
+    )
+
+    # ==============================================
+    #       Add likelihood model(s)
+    # ==============================================
+
+    # add the likelihood model to the problem
+    problem.add_likelihood_model(
+        GaussianLikelihoodModel(
+            prms_def="sigma",
+            experiment_name=experiment_name,
+            model_error="additive",
+            name="SimpleLikelihoodModel",
+        )
+    )
+
+    # give problem overview
+    problem.info()
+
+    # ==============================================
+    #        Export knowledge graph
+    # ==============================================
+
+    # create the knowledge graph and print it to file
+    #dir_path = os.path.dirname(__file__)
+    dir_path = calibrated_data_path
+    basename_owl = os.path.basename(__file__).split(".")[0] + ".owl"
+    knowledge_graph_file = os.path.join(dir_path, basename_owl)
+    export_knowledge_graph(problem, knowledge_graph_file, data_dir=dir_path)
```

Comment:
Where is that stored (this is hardcoded and for multiple data sets this is overwritten. I would also assume that it might be advantageous to return the graph instead of the file, such that writing to a file could actually be done outside in the doit script rather than the library (or you would have to give the file location as input).

Reply:
I am not sure I understand this comment. Yes you are right, it generates an .owl file and henceforth the KG for a given dataset, but for multiple dataset. But cant be move forward with this limited functionality for now?

Reply:
If we loop in our simple usecase over different tests, then there is no way we could keep these different KG? Or attach it to the global KG?